dts/bindings/dma/nxp,mcux-edma-v4.yaml

```diff
@@ -12,3 +12,16 @@ properties:
     type: boolean
     description: |
       If the SoCs don't have a separate interrupt id for error IRQ.
+  dma-channels:
```

Comment:
who else is using this binding, is making this required true in this commit going to break bisectibility ?